### DISCUSSION TITLE
poetry install not working

### DISCUSSION THREAD

**Discussion by sarang-tech-23** [BeanieODM/beanie#627] [Question]
Hi all, 
- I was trying to setup up Beanie development environment on my machine, 
- I took the latest pull of repo and downloaded **potery(version 1.5.1)** using brew on my mac M1.
- but when I ran the  **_poetry install_** command, it is throwing the following error 
- Error:- `[tool.poetry] section not found in /Users/sarangpatil/Documents/developer/my-repos/beanie/pyproject.toml`
- I checked that the **_pyproject.toml_** file was just changed few weeks ago and in that change **_[tool.poetry]_** was changed to **_[project]_**.
- I cant see any support for **_[project]_** in the official documentation of **_poetry_**
- Could someone please assist me with this? Thank you.

**Comment by roman-right** [comment]
Hi! Sorry for the confusion. Beanie is not a poetry project since 1.20. I've updated the respective documentation.

**Reply by sarang-tech-23** [reply]
Thanks, it is working now.
